Makefile

```diff
@@ -149,13 +149,13 @@ generate_yaml_test: kubectl
 ############################################################
 build: manager clusterctl
 
-manager: depend generate check
```

Comment:
why did you remove `depend generate check`?

Reply:
The related steps lasts for long time in building a binary. I think it is not necessary for testing a simple change in code for the binary.

Reply:
We cannot delete this, as the ` depend generate check` can make sure that we can always have correct format, and updated vendor files.

Reply:
update code to make old target keep as is.

add new short target for quick build to accelerate working efficiency.